WLC/code_executor/code_executor.py

```diff
@@ -0,0 +1,28 @@
+from ..code_executor.haskell_executor import HaskellExecutor
+from ..code_executor.python_executor import PythonExecutor
+
+
+class CodeExecutor:
+    def __init__(self, language="python3", ip="", port=""):
+        if language == "haskell":
+            self.executor = HaskellExecutor(ip, port)
+        else:
```

Comment:
Check the` language == 'python3'` for homogeneity?

Reply:
I wanted it to default to PythonExecutor even if an invalid language is provided.

Reply:
I would advise against that. This way we can end up burying some bugs but it's fine right now I guess.